### PR TITLE
Log ingestion lifecycle

### PR DESCRIPTION
**Pull request by mikeminutillo**
Adds additional logging around the audit ingestion component lifecycle.

### CODE REVIEW

**Review by WilliamBZA — APPROVED** [review]
@mikeminutillo I added a few more statements, can you give them a once over too?

File: src/ServiceControl.Audit/Auditing/AuditPersister.cs [hunk 1]
```diff
@@ -74,21 +74,37 @@ public async Task<IReadOnlyList<MessageContext>> Persist(List<MessageContext> co
                             RecordKnownEndpoints(receivingEndpoint, knownEndpoints, processedMessage);
                         }
 
+                        if (Logger.IsDebugEnabled)
+                        {
+                            Logger.Debug($"Adding message for bulk storage");
```
Comment (mikeminutillo) [inline src/ServiceControl.Audit/Auditing/AuditPersister.cs:79]: Should we explicitly say "Audit Message"?

Comment (WilliamBZA) [inline src/ServiceControl.Audit/Auditing/AuditPersister.cs:79]:
```suggestion
                            Logger.Debug($"Adding audit message for bulk storage");
```

File: src/ServiceControl.Audit/Auditing/AuditPersister.cs [hunk 2]
```diff
@@ -74,21 +74,37 @@ public async Task<IReadOnlyList<MessageContext>> Persist(List<MessageContext> co
...
+                        }
                         await bulkInsert.StoreAsync(processedMessage).ConfigureAwait(false);
                         storedContexts.Add(context);
                     }
                     else if (context.Extensions.TryGet(out SagaSnapshot sagaSnapshot))
                     {
+                        if (Logger.IsDebugEnabled)
+                        {
+                            Logger.Debug($"Adding message for bulk storage");
```
Comment (WilliamBZA) [inline src/ServiceControl.Audit/Auditing/AuditPersister.cs:88]:
```suggestion
                            Logger.Debug($"Adding SagaSnapshot message for bulk storage");
```

**Pull request merged**